docs/glossary/entry-chunk.md

```diff
@@ -0,0 +1,73 @@
+# Entry Chunk
+
+An entry chunk is created, because we need to output a JavaScript file for:
```

Comment:
Should we explain what an entry chunk is first?

Reply:
It should be included in the `Chunk` term.